app/src/main/res/values/strings.xml

```diff
@@ -46,4 +46,24 @@
   <string name="bottom_nav_trips">Trips</string>
   <string name="bottom_nav_profile">Profile</string>
 
+  <string name="image_rockies_content_description">Image of woman standing on a pier by a lake in the mountains</string>
```

Comment:
As a screen reader will already identify something is an image, there's no need to repeat that information.
Sometimes it's useful to say "photo of..." or "illustration of..." to convey the type of image.

```suggestion
  <string name="image_rockies_content_description">Woman standing on a pier by a lake in the mountains</string>
```

Reply:
That is what I thought as well, but when testing it didn't read out that it was an image :(